pydemolib/nessiedemo/demo.py

```diff
@@ -131,6 +131,16 @@ def get_iceberg_version(self: T) -> str:
         """Get the Iceberg version defined in the versions-dictionary."""
         return self.__iceberg_version
 
+    def get_iceberg_download_url_for_jar(self: T, iceberg_component: str) -> str:
```

Comment:
Would prefer to have this one protected (rename to `_get_iceberg_download_url_for_jar`)